lib/cldr/ldml/attribute.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Cldr
+  module Ldml
+    class Attribute
+      class << self
+        def parse(attlist, comments)
+          element_name, attribute_name, type, mode = attlist.strip.split(/\s+/).map(&:to_sym)
+          status = comments.find { |comment| comment == :VALUE || comment == :METADATA } || :DISTINGUISHING
+          deprecated = comments.any? { |comment| comment == :DEPRECATED }
+          Attribute.new(element_name, attribute_name, type, mode, status, deprecated)
+        end
+      end
+
+      attr_reader :element_name, :attribute_name, :type, :mode
+
+      def initialize(element_name, attribute_name, type, mode, status, deprecated) # rubocop:disable Metrics/ParameterLists
```

Comment:
These names come from the CLDR Java code, which ultimately comes from the [Java's SAX API](www.saxproject.org/apidoc/org/xml/sax/ext/DeclHandler.html)